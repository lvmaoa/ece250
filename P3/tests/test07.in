load
load
load
load
load
load
size
empty
clear
size
empty
exit

