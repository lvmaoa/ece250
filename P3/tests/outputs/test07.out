success
success
success
success
success
success
number of words is: 274
empty 0
success
number of words is: 0
empty 1
